### ISSUE TITLE
Add a cloudformation export so the WebACL can be referenced in other stacks

### ISSUE THREAD

**Issue by matthewhembree** [aws-solutions/aws-waf-security-automations#216]
**Is your feature request related to a problem? Please describe.**
I'd like to be able to add the WebACL association through automation.

**Describe the feature you'd like**
Create a cloudformation export of the WebACL ARN that can be used in other stacks (and Terraform through `data.aws_cloudformation_export.my-webacl`).

e.g.:
```yaml
Type: AWS::WAFv2::WebACLAssociation
Properties: 
  ResourceArn: !Ref MyApiGateway.Arn
  WebACLArn: Fn::ImportValue:
    !Sub "${WAFStackName}-WebAclArn"
```

**Additional context**
Adding the example resource to my SAM deployments would ensure guaranteed attachment to all deployments (even feature/preview deployments).

**Comment by matthewhembree** [comment]
Fixed in #121

**Comment by aijunpeng** [comment]
This issue has been addressed in version >=4.0.0.
